src/schema/paddle.ts

```diff
@@ -231,6 +359,61 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
         items,
       };
     },
+    plusPricingMetadata: async (
+      _,
+      { variant = DEFAULT_PLUS_METADATA }: PlusMetadataArgs,
+      ctx: AuthContext,
+    ): Promise<PlusPricingMetadata[]> =>
+      await getPlusPricingMetadata(ctx.con, variant),
+    plusPricingPreview: async (_, __, ctx): Promise<PlusPricingPreview[]> => {
+      const gb = getUserGrowthBookInstace(ctx.userId);
+      const variant = gb.getFeatureValue(
+        PLUS_FEATURE_KEY,
+        DEFAULT_PLUS_METADATA,
+      );
+      const metadata = await getPlusPricingMetadata(ctx.con, variant);
+      const ids = metadata.map(({ idMap }) => idMap.paddle);
+      const preview = await getPlusPricePreview(ctx, ids);
+
+      // consolidate the preview data and metadata
+      const consolidated = metadata.map((meta) => {
+        const item = preview.details.lineItems.find(
+          (item) => item.price.id === meta.idMap.paddle,
+        );
+
+        if (!item) {
+          return null;
+        }
+
+        const isOneOff = !item.price.billingCycle?.interval;
+        const isYearly = item.price.billingCycle?.interval === 'year';
+        const duration =
+          isOneOff || isYearly
+            ? SubscriptionCycles.Yearly
+            : SubscriptionCycles.Monthly;
+        const baseAmount = getPriceFromPaddleItem(item);
+        const monthly = getPaddleMonthlyPrice(baseAmount, item);
+        const trialPeriod = item.price.trialPeriod;
+
+        return {
+          metadata: meta,
+          productId: item.price.productId,
+          price: {
+            monthly,
+            amount: baseAmount,
+            formatted: item.formattedTotals.total,
+          },
+          currency: {
+            code: preview.currencyCode,
+            symbol: item.formattedTotals.total.replace(/\d|\.|\s|,/g, ''),
```

Comment:
Consider making it a named regex to know what it does :)

Reply:
Moved! 🚀

Reply:
don't see its moved?

Reply:
Right. I moved the other one that used the same regex. Should be update now 🙏